packages/web/src/components/premium-content-purchase-modal/components/PurchaseDetailsPage.tsx

```diff
@@ -83,7 +49,7 @@ export type PurchaseDetailsPageProps = {
 }
 
 export const PurchaseDetailsPage = ({
-  currentBalance = zeroBalance(),
+  currentBalance,
```

Comment:
Curious why this got removed? I think this was to prevent problems with users who don't have a user bank yet.

Reply:
i moved it as a default arg in the `getPurchaseSummaryValues` fn. buttt i can see an argument for keeping it here instead, makes more sense conceptually. wdyt?

Reply:
Oh I missed that. Probably fine, just might complicate things if we need the current balance for any other reason inside this component.